Bolt 02: Limited the scope in which the funding TX MUST NOT  be broadcasted (#583)
lightningnetwork/lightning-rfc: 02-peer-protocol.md

The sender MUST set:
  - `temporary_channel_id` the same as the `temporary_channel_id` in the `open_channel` message.
  - `funding_txid` to the transaction ID of a non-malleable transaction,

>     - and MUST NOT broadcast this transaction before a valid `funding_signed` message was received. 

Reviewer: ```suggestion
    - and MUST NOT broadcast this transaction until a valid `funding_signed` message is received. 
```
